pkgs/by-name/mm/mmdebstrap/package.nix

```diff
@@ -0,0 +1,126 @@
+{ lib
+, stdenv
+, fetchFromGitea
+
+, glibc
+, help2man
+, makeWrapper
+, patsh
+
+, apt
+, coreutils
+, dpkg
+, fakechroot
+, fakeroot
+, findutils
+, gnupg
+, gnutar
+, perl
+, python3
+, shadow
+, util-linux
+
+, mmdebstrap
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mmdebstrap";
+  version = "1.4.0";
+
+  src = fetchFromGitea {
+    domain = "gitlab.mister-muffin.de";
+    owner = "josch";
+    repo = "mmdebstrap";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-ulW+zCYVMqi6wzqPYYlTsKZqgW9DmClTFJdEMRY4yiI=";
+  };
+
+  nativeBuildInputs = [
+    glibc
+    help2man
+    makeWrapper
+    patsh
+  ];
+
+  buildInputs = [
+    perl
+    python3
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+
+    patchShebangs ./tarfilter
+    patsh -f ./mmdebstrap-autopkgtest-build-qemu -s ${builtins.storeDir}
+
+    pod2man ./mmdebstrap > ./mmdebstrap.1
+    pod2man ./mmdebstrap-autopkgtest-build-qemu > ./mmdebstrap-autopkgtest-build-qemu.1
+    help2man --no-info --name "filter a tarball like dpkg does" --version-string="${finalAttrs.version}" ./tarfilter > ./tarfilter.1
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin/
+    mv mmdebstrap $out/bin/
+    mv mmdebstrap-autopkgtest-build-qemu $out/bin/
+    mv tarfilter $out/bin/mmtarfilter
+
+    mkdir -p $out/man/man1/
+    mv mmdebstrap.1 $out/man/man1/
+    mv mmdebstrap-autopkgtest-build-qemu.1 $out/man/man1/
+    mv tarfilter.1 $out/man/man1/mmtarfilter.1
```

Comment:
Manpages are installed via `installManPage` from the `installShellFiles` package.